pkg/maintainer/relay.go

```diff
@@ -0,0 +1,54 @@
+package maintainer
+
+import (
+	"context"
+	"time"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+var logger = log.Logger("keep-maintainer-relay")
+
+func newRelay(
+	ctx context.Context,
+	btcChain bitcoin.Chain,
+	relayChain RelayChain,
+) *Relay {
+	relay := &Relay{
+		btcChain:   btcChain,
+		relayChain: relayChain,
+	}
+
+	go relay.startControlLoop(ctx)
+
+	return relay
+}
+
+// Relay is the part of maintainer responsible for maintaining the state of
+// the relay on-chain contract.
+type Relay struct {
+	btcChain   bitcoin.Chain
+	relayChain RelayChain
```

Comment:
I think we can make it simpler:
```suggestion
	chain RelayChain
```